src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -845,9 +810,8 @@ uct_cuda_copy_md_open(uct_component_t *component, const char *md_name,
     md->config.alloc_whole_reg   = config->alloc_whole_reg;
     md->config.max_reg_ratio     = config->max_reg_ratio;
     md->config.pref_loc          = config->pref_loc;
+    md->config.fabric_supported  = config->enable_fabric;
```

Comment:
rename md->config.fabric_supported to md->config.enable_fabric and define it as a ternary value instead of int